QueryFilter/QueryFilter.php

```diff
@@ -201,8 +211,8 @@ private function getSearchBy(ConfigInterface $config): array
     {
         // Get basic search by
         $searchBy = $config->getRequest()->isSimple()
-            ? $this->getSimpleSearchBy($config->getSearchAllowedCols(), $config->getRequest()->getQuery())
-            : $this->getFullSearchBy($config->getSearchAllowedCols(), $config->getRequest()->getQuery());
+            ? $this->getSimpleSearchBy($config->getSearchAllowedCols(), $config->getRequest()->getQuery(), $config->isStrictColumns())
```

Comment:
Would you mind to add a couple of tests to QueryFilterTest.php to cover the new option?

Reply:
Sure, test cases added.